src/node-types.ts

```diff
@@ -137,25 +137,25 @@ export type TextNode = {
 
 export type StarNode = {
   name: 'star';
-  contents: FragmentNode[];
+  contents: (TextNode | CommentNode | TagNode)[];
```

Comment:
Since this is a breaking change anyway, is there sufficient reason to continue supporting comments and tags inside `*`, ``` ` ```, and `~` formatting wrappers?

Reply:
This PR is breaking in the sense of requiring integration work in ecmarkup, but not in the sense of requiring any specifications to change. Whereas prohibiting tags inside of `*` would actually be a breaking change [for at least Temporal](https://github.com/tc39/proposal-temporal/blob/d05a18b025fdf0bfc857b2bcc4158edaf9bb910a/spec/instant.html#L72). Since I've already written the code necessary to support the more complicated types there's not that much to be gained by restricting these and forcing Temporal to adapt, I think.

Reply:
Fair enough. I've opened https://github.com/tc39/proposal-temporal/pull/2494 against Temporal regardless, and don't see any offending text in either ecma262 or ecma402. It might also make a good linter check, but that's obviously not necessary.

Reply:
>  it should probably be documented at

It is, I think? The constraint on `_` is "cannot contain whitespace or other formatting characters", whereas the constraint on `*` is "cannot contain asterisks", for example.